tests/prover-e2e/Makefile

```
@@ -0,0 +1,35 @@
+.PHONY: clean setup_db test_tool
+
+GOOSE_CMD?=goose
+BEGIN_BLOCK?=10405848
+END_BLOCK?=10405858
+
+all: setup_db test_tool import_data
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)